wagtail/search/tests/test_elasticsearch6_backend.py

```diff
@@ -0,0 +1,801 @@
+# -*- coding: utf-8 -*-
+import datetime
+import json
+
+import mock
+from django.db.models import Q
+from django.test import TestCase
+from elasticsearch.serializer import JSONSerializer
+
+from wagtail.tests.search import models
+from wagtail.search.backends.elasticsearch6 import Elasticsearch6SearchBackend
+
+from .elasticsearch_common_tests import ElasticsearchCommonSearchBackendTests
+from .test_backends import BackendTests
+
+
+class TestElasticsearch6SearchBackend(BackendTests, ElasticsearchCommonSearchBackendTests, TestCase):
```

Comment:
Currently failing with `TypeError: Cannot create a consistent method resolution
order (MRO) for bases TestCase, ElasticsearchCommonSearchBackendTests, BackendTests` - I think `BackendTests` needs to be removed (since `ElasticsearchCommonSearchBackendTests` already inherits from it).